### DISCUSSION TITLE
How to increase timeout duration for AWS_IO_TLS_NEGOTIATION_TIMEOUT

### DISCUSSION THREAD

**Discussion by dsoni768** [aws/aws-iot-device-sdk-cpp-v2#699] [Q&A] [answered]
We have custom linux based hardware on which AWS FleetWise application is being deployed.
We are getting below error

**_[Thread: 1440] [2024-03-27T09:18:12.924Z] [ERROR] [AwsIotConnectivityModule.cpp:223] [operator()()]: [Connection failed with error code 1067: aws-c-io: AWS_IO_TLS_NEGOTIATION_TIMEOUT, Channel shutdown due to tls negotiation timeout]_**

I wanted to increase the timeout duration for TLS negotiatio to check if the error is related to timeout only.

**Comment by bretambrose** [comment, marked as answer]
At present, there is not a way to increase the base tls negotiation timeout (10 seconds I think).
